scapy/fields.py

```diff
@@ -1165,6 +1165,16 @@ def __rshift__(self, other):
     def __nonzero__(self):
         return bool(self.value)
     __bool__ = __nonzero__
+    def __div__(self, other):
```

Comment:
It has no meaning to divide a flag value by something. Could you remove this part?